toolkit/scripts/toolchain.mk

```diff
@@ -171,59 +171,72 @@ $(final_toolchain): $(raw_toolchain) $(STATUS_FLAGS_DIR)/build_toolchain_srpms.f
 .SILENT: $(toolchain_rpms)
 
 ifeq ($(REBUILD_TOOLCHAIN),y)
+# We know how to build this archive from scratch
+selected_toolchain_archive = $(final_toolchain)
+else
+# This may be an empty string, that is fine. If its empty we will try to use online packages
+selected_toolchain_archive = $(TOOLCHAIN_ARCHIVE)
+endif
+
+# We will have three options at this point:
+# 1) REBUILD_TOOLCHAIN: yes										-> Rebuild a toolchain from scratch and pace it into $(final_toolchain)
```

Comment:
can you align the spacing here? also typo: pace -> place

Reply:
Oh, interesting, GitHub is assuming a different tab size for this file. Makefiles generally use tabs everywhere, but I'll convert the spacing in those lines to spaces since they are comments.